Makefile

```
@@ -240,13 +235,10 @@ flakehunter-parallel:
 # =============
 # FUZZING
 # =============
-fuzz-build: $(GOFUZZ_BUILD_BIN) $(GOFUZZ_DEP_BIN)
-	@$(call print, "Creating fuzz harnesses for packages '$(FUZZPKG)'.")
-	scripts/fuzz.sh build "$(FUZZPKG)"
 
-fuzz-run: $(GOFUZZ_BIN)
+fuzz: $(GOFUZZ_BIN)
```

Since there is a directory called `fuzz`, this target won't build.

We should also remove the `GOFUZZ_BIN` usage.
```suggestion
fuzz-run:
```